samples/mqtt/receiver/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net"
+
+	cemqtt "github.com/cloudevents/sdk-go/protocol/mqtt_paho/v2"
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+	"github.com/eclipse/paho.golang/paho"
+)
+
+func main() {
+	conn, err := net.Dial("tcp", "127.0.0.1:1883")
+	if err != nil {
+		log.Fatalf("failed to connect to mqtt broker: %s", err.Error())
+	}
+	clientConfig := &paho.ClientConfig{
+		ClientID: "receiver-client-id",
+		Conn:     conn,
+	}
+	cp := &paho.Connect{
+		KeepAlive:  30,
+		CleanStart: true,
+	}
+
+	p, err := cemqtt.New(context.TODO(), clientConfig, cp, "test-topic", []string{"test-topic"}, 0, false)
```

Comment:
question: is sender topic here required, or would `""` be ok?

Reply:
The senderTopic isn't necessary for the receiver, `""` is ok.